<launch>
    <arg name="topic_input_cloud" />
    <arg name="topic_voxels" />
    <arg name="filter_by_score_enabled" />
    <arg name="filter_by_score_minimum_score" />
    <arg name="sor_enabled" />
    <arg name="sor_mean_k" />
    <arg name="sor_stddev_mul_thresh" />

    <node name="boxing" pkg="boxing" type="boxing">
        <param name="topic_input_cloud" type="string" value="$(arg topic_input_cloud)" />
        <param name="topic_voxels" type="string" value="$(arg topic_voxels)" />

        <param name="filter_by_score_enabled" type="bool" value="$(arg filter_by_score_enabled)" />
        <param name="filter_by_score_minimum_score" type="double" value="$(arg filter_by_score_minimum_score)" />

        <param name="sor_enabled" type="bool" value="$(arg sor_enabled)" />
        <param name="sor_mean_k" type="double" value="$(arg sor_mean_k)" />
        <param name="sor_stddev_mul_thresh" type="double" value="$(arg sor_stddev_mul_thresh)" />
    </node>
</launch>
